### DISCUSSION TITLE
AlienFan GUI - CPU Mode AC and Battery

### DISCUSSION THREAD

**Discussion by crmsnswrdsmn** [T-Troll/alienfx-tools#358] [Q&A] [answered]
What do "Aggressive, Enabled, Off, Efficient, and Efficient Aggressive" mean in CPU Mode AC and Battery in AlienFan GUI?

I can't seem to find an answer/explanation about these settings, either here on github or reddit :(

Thanks!

**Comment by T-Troll** [comment]
It controls a hidden setting into Windows Power Plan - the slope of frequency acceleration (close to EPP into Throttle Stop).
This is not so important for Intel CPU, but provide nice boost for AMD.

**Reply by crmsnswrdsmn** [reply]
Thank you. But what does Aggressive, Enabled, Off, Efficient, and Efficient Aggressive mean? Thanks

**Reply by T-Troll** [reply]
Off - default slope
Aggressive - faster gain
Efficient - lower gain (for battery power)
EA - something between off and efficient.

You can change it and run benchmark. For me, Aggressive give a little (1%) boost, Efficient - about 1% lower scope.

**Reply by crmsnswrdsmn** [reply]
I have an m15R5 with Ryzen 5900HX, and I am mainly running on AC power (also set in BIOS).. What do you personnally recommend?

**Reply by T-Troll** [reply, marked as answer]
It should be more for Ryzen, but i don't know.
You can google stats for AMD CPU using these settings - not only my tool have it!

**Reply by crmsnswrdsmn** [reply]
Thanks!
